src/utils/rpc.ts

```diff
@@ -1,8 +1,11 @@
 import { Contract, providers } from 'ethers'
 
-const PROVIDER = 'https://xdai.fairdatasociety.org/'
+const PROVIDER = 'https://gno.getblock.io/mainnet/?api_key=d7b92d96-9784-49a8-a800-b3edd1647fc7'
```

Comment:
You may not want to commit your `api_key` to the repo.

Reply:
Intentional for now :)